lib/smart_proxy_openscap/openscap_import_api.rb

```diff
@@ -0,0 +1,36 @@
+module Proxy::OpenSCAP
+  class ImportApi < ::Sinatra::Base
+    include ::Proxy::Log
+    helpers ::Proxy::Helpers
+    authorize_with_trusted_hosts
```

Comment:
I see. If people run smart proxy on other than foreman host, they might be missing trusted host setup I think, it should be documented somewhere.